spec/services/ingest_iati_activities_spec.rb

```diff
@@ -3,14 +3,25 @@
 
 RSpec.describe IngestIatiActivities do
   describe "#call" do
-    it "creates 36 new projects" do
+    it "creates 35 new projects" do
```

Comment:
I am struggling to see why this has changed when the fixture has not?

Reply:
This is a good point. I haven't explained why this happened! I have subsequently changed this back to 36 in the [more recent work to change how the tool deals with programmes](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/411). Once that's merged I'll fix up this pull request so that it should disappear 👍